src/instructor-toolbar/masquerade-widget/MasqueradeWidget.jsx

```diff
@@ -120,7 +120,7 @@ class MasqueradeWidget extends Component {
     return (
       <div className="flex-grow-1">
         <div className="row">
-          <span className="col-auto col-form-label pl-3">View this course as:</span>
+          <span className="col-auto col-form-label pl-3">{this.props.intl.formatMessage(messages.titleViewCourseIn)}</span>
```

Comment:
Nit: prefer the component form in a JSX context like this:
```suggestion
          <span className="col-auto col-form-label pl-3"><FormattedMessage {...messages.titleViewCourseIn} /></span>
```